Vagrantfile

```diff
@@ -1,9 +1,22 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
 
 Vagrant.configure(2) do |config|
+
   config.vm.box = "ubuntu/trusty64"
```

Comment:
We may want to consider that other CFPB Vagrant setups use CentOS rather than Ubuntu. Ubuntu is here for continuity with the upstream 18F regulations-bootstrap.